roles/openshift_cli/defaults/main.yml

```diff
@@ -8,4 +8,4 @@ system_images_registry: "{{ system_images_registry_dict[openshift_deployment_typ
 openshift_use_crio_only: False
 
 l_is_system_container_image: "{{ openshift_use_master_system_container | default(openshift_use_system_containers | default(False)) | bool }}"
-l_use_cli_atomic_image: "{{ (openshift_use_crio_only | bool) or (l_is_system_container_image | bool) }}"
+l_use_cli_atomic_image: "{{ (openshift_use_crio_only | bool and openshift_crio_use_system_container | bool) or (l_is_system_container_image | bool) }}"
```

Comment:
openshift_crio_use_system_container is not used anymore, need to change to newer openshift_crio_use_system_container var.

Can define default as openshift_crio_use_system_container: False in this file.

Reply:
Good catch.

Reply:
@michaelgugino how about this:
```
l_use_cli_atomic_image: "{{ (openshift_use_crio_only | bool and not openshift_crio_use_rpm | bool) or (l_is_system_container_image | bool) }"
```

Reply:
That looks okay, need to define openshift_crio_use_rpm: False in this file..

Reply:
Sounds good.